packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -136,6 +128,28 @@ function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
 					} }
 					dropdownMenuProps={ dropdownMenuProps }
 				>
+					<ToolsPanelItem
+						isShownByDefault
+						hasValue={ () => averageReadingSpeed !== undefined }
```

Comment:
```suggestion
						hasValue={ () =>
							averageReadingSpeed !==
							DEFAULT_AVERAGE_READING_SPEED
						}
```

Here we need to check if it is NOT the default value, otherwise the reset button will always be displayed.

It would be nice to define `const DEFAULT_AVERAGE_READING_SPEED = 189` somewhere.